### PR TITLE
M3 5250: Clean up Tables CMR

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Replace old Table-related components with the new CMR version and rename the file wherever it's being imported

## How to test

Make sure all the components that were updated all look good

### CODE REVIEW

**Comment by bnussman** [comment]
I'm seeing some horizontal scroll bars on some tables on `/domains/{id}` that were not there before

Screen Shot 2021-06-16 at 9 57 09 AM

**Comment by bnussman** [comment]
Found a slight dark mode bug that was introduced on the Linode graph's table header 

Screen Shot 2021-06-16 at 1 39 46 PM

**Comment by bnussman** [comment]
Seeing the same overflow problem with the backups table.

Screen Shot 2021-06-17 at 9 23 54 AM

**Review by bnussman — CHANGES_REQUESTED** [review]

File: packages/manager/src/features/Longview/LongviewDetail/DetailTabs/Processes/ProcessesTable.tsx [hunk 1]
```diff
@@ -4,25 +4,17 @@ import { makeStyles, Theme } from 'src/components/core/styles';
...
-import TableRowError_CMR from 'src/components/TableRowError/TableRowError_CMR';
-import TableRowLoading_PreCMR from 'src/components/TableRowLoading';
-import TableRowLoading_CMR from 'src/components/TableRowLoading/TableRowLoading_CMR';
+import Table from 'src/components/Table';
+import TableCell from 'src/components/TableCell';
+import TableRow from 'src/components/TableRow';
+import TableRowEmptyState from 'src/components/TableRowEmptyState';
+import TableRowError from 'src/components/TableRowError';
+import TableRowLoading from 'src/components/TableRowLoading';
 import TableSortCell_PreCMR from 'src/components/TableSortCell';
```
[the remark this answers is not in the document]
Comment (tiffwong) [reply packages/manager/src/features/Longview/LongviewDetail/DetailTabs/Processes/ProcessesTable.tsx:13]: Nice catch!

**Review by DevDW — COMMENTED** [review]
I still need to comb through the code and the app a bit more, but I haven't noticed any UI bugs so far. Banks raised a good point about the `linkTarget` piece with the Events though; we'll definitely want to restore that functionality.

File: packages/manager/src/features/Events/EventRow.tsx [hunk 2]
```diff
@@ -102,7 +89,6 @@ export const Row: React.FC<RowProps> = (props) => {
 
   return (
     <TableRow
-      rowLink={entityId ? undefined : (linkTarget as any)}
```
[the remark this answers is not in the document]
Comment (tiffwong) [reply packages/manager/src/features/Events/EventRow.tsx:105]: This should be fixed now 😅

**Review by bnussman — APPROVED** [review]
Everything throughout the app looks good to me! More 👀 on this one might be nice just to check for things I may have missed

**Review by DevDW — CHANGES_REQUESTED** [review]
I have a little bit more to test, but I did find a bug in `DomainRecordActionMenu`

File: packages/manager/src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackupActionMenu.tsx [hunk 3]
```diff
@@ -13,49 +12,38 @@ interface Props {
...
-      <ActionMenu
-        createActions={this.createActions()}
-        ariaLabel={`Action menu for Backup ${this.props.backup.label}`}
-      />
-    );
-  }
-}
+  const actions: Action[] = [
+    {
+      title: 'Restore to Existing Linode',
```
Comment (DevDW) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackupActionMenu.tsx:26]: side note: clicked this and noticed the hover color of the "Cancel" button in the drawer is odd

Screen Shot 2021-06-21 at 5 45 09 PM

**Review by DevDW — APPROVED** [review]
LGTM now!

File: packages/manager/src/components/TableRow/TableRow.test.tsx [hunk 4]
```diff
@@ -27,7 +21,7 @@ const props: CombinedProps = {
 };
 
 describe('TableRow component', () => {
-  it('calls history.push with the given rowLink', () => {
+  it.skip('calls history.push with the given rowLink', () => {
```
Comment (HanaXu) [inline packages/manager/src/components/TableRow/TableRow.test.tsx:24]: Do we still need this test if we removed `rowLink`?

**Review by HanaXu — APPROVED** [review]
LGTM! Left a comment about the rowLink test

File: packages/manager/src/components/TableRow/TableRow.test.tsx (hunk 4 above)
Comment (tiffwong) [reply packages/manager/src/components/TableRow/TableRow.test.tsx:24]: Probably not but I wasn't sure how to fix the test otherwise lol

**Review by HanaXu — COMMENTED** [review]
Actually one more thing, can we fix this warning?

image

**Pull request merged**